src/components/NewCommentForm/NewCommentForm.tsx

```diff
@@ -0,0 +1,105 @@
+import './NewCommentForm.scss';
+import React, { FormEvent, useState } from 'react';
+import { Comment } from '../../types';
+
+type Props = {
+  commentsList: Comment[] | undefined;
+  setCommentsList: (arg:Comment[]) => void;
+};
+
+export const NewCommentForm: React.FC<Props> = ({
+                                                  commentsList, setCommentsList,
+                                                }) => {
+  const [yourname, setYourname] = useState('');
+  const [youremail, setYouremail] = useState('');
+  const [yourcomment, setYourcomment] = useState('');
```

Comment:
You can also combine these states into one.
For example:
```
const [userCommentData, setUserCommentData] = useState<TypeForState>({
name: '',
email: '',
comment: ''
});
```
To change this state, it is best to use the functional setState:
```
const changeUserCommentData = (value: string, field: keyof TypeForState) {[
setUserCommentData(prevUserCommentData => ({...prevUserCommentData, [field]: value}))
}
```